cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -39,7 +39,10 @@ CREATE INDEX IF NOT EXISTS geom_index
 
 CREATE TABLE IF NOT EXISTS water_systems(
     pws_id varchar(255) NOT NULL,
+    pws_name varchar(255),
```

Comment:
Changes to tables need to be in an `ALTER TABLE` statement. Otherwise these changes will be skipped for people who already have this table (like I did). 

Especially since we'll have real data in here and won't want to drop the tables and re-create them all the time.